tests/foreman/api/test_provisioning.py

```diff
@@ -187,126 +185,125 @@ def test_verify_bugzilla_1114640(self, target_sat):
 class TestOperatingSystem:
     """Tests for operating systems."""
 
-    @pytest.mark.tier1
-    @pytest.mark.parametrize('name', **parametrized(valid_data_list()))
-    def test_positive_create_with_name(self, name, target_sat):
```

Comment:
Can you keep this parametrized test for OS create with names from valid_data_list ?

Reply:
Updated.